### ISSUE TITLE
[LGTM] Code analysis: cs/useless-assignment-to-local

### ISSUE THREAD

**Issue by jthelin** [jthelin/ServerHost#81]
LGTM code analysis reported the following issue:

**This assignment to unused is useless, since its value is never read.**

An assignment to a local variable that is not used later on, or whose value is always overwritten, has no effect.

https://lgtm.com/rules/1506093386171/

https://github.com/jthelin/ServerHost/blob/ca3749fd13e607fba4cb0a902cef6e83fa2350bb/ServerHost/ServerHost.cs#L171

https://lgtm.com/projects/g/jthelin/ServerHost/snapshot/d7c80fd48f0dcabb221cd84f92c7b16d46e028da/files/ServerHost/ServerHost.cs?sort=name&dir=ASC&mode=heatmap#xbbc6213c3d4c8f26:1

**Comment by jthelin** [comment]
SARIF format description of the issue:

```
{
    "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
    "version": "2.1.0",
    "runs": [
        {
            "tool": {
                "driver": {
                    "name": "LGTM.com",
                    "productSuite": "Semmle",
                    "version": "1.23.0-SNAPSHOT",
                    "rules": [
                        {
                            "id": "com.lgtm/csharp-queries:cs/useless-assignment-to-local",
                            "name": "com.lgtm/csharp-queries:cs/useless-assignment-to-local",
                            "shortDescription": {
                                "text": "Useless assignment to local variable"
                            },
                            "fullDescription": {
                                "text": "An assignment to a local variable that is not used later on, or whose value is always overwritten, has no effect."
                            },
                            "defaultConfiguration": {},
                            "properties": {
                                "tags": [
                                    "maintainability",
                                    "external/cwe/cwe-563"
                                ],
                                "kind": "problem",
                                "precision": "very-high",
                                "problem.severity": "warning"
                            }
                        }
                    ]
                }
            },
            "artifacts": [
                {
                    "location": {
                        "uri": "ServerHost/ServerHost.cs",
                        "uriBaseId": "%SRCROOT%",
                        "index": 0
                    }
                }
            ],
            "results": [
                {
                    "ruleId": "com.lgtm/csharp-queries:cs/useless-assignment-to-local",
                    "ruleIndex": 0,
                    "message": {
                        "text": "This assignment to [unused](1) is useless, since its value is never read."
                    },
                    "locations": [
                        {
                            "physicalLocation": {
                                "artifactLocation": {
                                    "uri": "ServerHost/ServerHost.cs",
                                    "uriBaseId": "%SRCROOT%",
                                    "index": 0
                                },
                                "region": {
                                    "startLine": 171,
                                    "startColumn": 24,
                                    "endLine": 171,
                                    "endColumn": 55,
                                    "charOffset": 7047,
                                    "charLength": 31
                                }
                            }
                        }
                    ],
                    "partialFingerprints": {
                        "primaryLocationLineHash": "bbc6213c3d4c8f26:1",
                        "primaryLocationStartColumnFingerprint": "7"
                    },
                    "relatedLocations": [
                        {
                            "id": 1,
                            "physicalLocation": {
                                "artifactLocation": {
                                    "uri": "ServerHost/ServerHost.cs",
                                    "uriBaseId": "%SRCROOT%",
                                    "index": 0
                                },
                                "region": {
                                    "startLine": 171,
                                    "startColumn": 24,
                                    "endLine": 171,
                                    "endColumn": 30,
                                    "charOffset": 7047,
                                    "charLength": 6
                                }
                            },
                            "message": {
                                "text": "unused"
                            }
                        }
                    ]
                }
            ],
            "columnKind": "utf16CodeUnits",
            "properties": {
                "semmle.formatSpecifier": "2.1.0",
                "semmle.sourceLanguage": "csharp"
            }
        }
    ]
}
```

**Comment by jthelin** [comment]
Fixed by PR #82
